app/src/main/java/com/kickstarter/viewmodels/SearchViewModel.java

```diff
@@ -30,6 +32,9 @@
 
     /** Call when text changes in search box. */
     void search(final @NonNull String s);
+
+    // FIXME: document
+    void tappedProject(final @NonNull Project project);
```

Comment:
we like to do noun-verb for our inputs, i.e. `projectTapped`
also since we're in android land we still use `clicked`, i.e. `projectClicked` (sry not sry @mbrandonw)